stdlib/public/Concurrency/Actor.cpp

```diff
@@ -48,9 +48,12 @@
 #include <sys/syscall.h>
 #endif
 
-#if HAVE_PTHREAD_H
+#if __has_include(<pthread.h>)
```

Comment:
This should not be changed.

Reply:
The trouble is that this isn't defined on other platforms. So, the alternative here is to drive this statically from the build system...